### ISSUE TITLE
Why do I get very low initial bbox values when using the COCO dataset to fine-tune SAM3？

### ISSUE THREAD

**Issue by Troy0630** [facebookresearch/sam3#402]
<img width="2987" height="585" alt="Image" src="https://github.com/user-attachments/assets/bad0e00f-77df-4132-b652-0c209979945c" />
Why do I get very low initial bbox values when using the COCO dataset to fine-tune SAM3？
The pre-trained SAM3 should have relatively high detection capability on the COCO dataset, shouldn't it?

**Comment by rlewkowicz** [comment]
Wouldn't that indicate the opposite? That's your loss landscape. Not MAP. As you approach convergence, your loss should drop.

**Comment by WD7ang** [comment]
Which module did you fine-tune?
